arches/app/src/arches/themes/default.ts

```diff
@@ -0,0 +1,81 @@
+import { definePreset, palette } from "@primevue/themes";
+import Aura from "@primevue/themes/aura";
+
+const archesColors = Object.freeze({
+    blue: "#579ddb",
+    green: "#3acaa1",
+    red: "#f75d3f",
+});
+
+export const ArchesPreset = definePreset(Aura, {
+    primitive: {
+        arches: {
+            ...archesColors,
+            legacy: {
+                sidebar: "#2d3c4b",
+            },
+        },
+        green: palette(archesColors.green),
+        red: palette(archesColors.red),
+    },
+    semantic: {
+        primary: palette(archesColors.blue),
+        navigation: {
+            list: {
+                padding: "0",
+            },
+            item: {
+                padding: "1rem",
+            },
+            color: "{arches.legacy.sidebar}",
+        },
+        iconSize: "small",
+        // custom tokens
+        content: {
+            gap: "1rem",
+        },
+        fontFamily: "system-ui, sans-serif",
+    },
+    components: {
```

Comment:
IMO we should use overrides with a _very_ light touch. I get the splitter handle && agree. The other stuff not so much